src/test/java/uk/gov/companieshouse/overseasentitiesapi/controller/OverseasEntitiesControllerTest.java

```diff
@@ -613,6 +615,77 @@ void testCreatingANewSaveAndResumeSubmissionIsUnSuccessfulWhenValidationChecksFa
         assertEquals(HttpStatus.BAD_REQUEST.value(), response.getStatusCodeValue());
     }
 
+    @Test
+    void testGetSubmissionIsSucessful() throws SubmissionNotFoundException, SubmissionNotLinkedToTransactionException {
+        when(overseasEntitiesService.getSavedOverseasEntity(
+                transaction,
+                SUBMISSION_ID,
+                REQUEST_ID
+                )).thenReturn(ResponseEntity.ok().body(overseasEntitySubmissionDto));
+
+        var response = overseasEntitiesController.getSubmission(
+                transaction,
+                SUBMISSION_ID,
+                REQUEST_ID
+        );
+
+        assertEquals(HttpStatus.OK, response.getStatusCode());
+        assertNotNull(response.getBody());
+    }
+
+    @Test
+    void testGetSubmissionIsNotSucessfulWhenBadRequestIsReturned() throws SubmissionNotFoundException, SubmissionNotLinkedToTransactionException {
```

Comment:
i'd change the name of the test as it is testing the response when a SubmissionNotLinked exception is thrown, maybe something like
testGetSubmissionIsNotSuccessfulWhenSubmissionNotLinkedToTransaction